module-router.c

```diff
@@ -1102,11 +1132,24 @@ static pa_hook_result_t hook_callback_source_new(pa_core *c, pa_source *source,
 
 static pa_hook_result_t hook_callback_sink_input_new(pa_core *c, pa_sink_input_new_data *new_data, struct userdata *u) {
     ROUTER_FUNCTION_ENTRY;
+
     bool already_present = false;
 
     pa_assert(c);
     pa_assert(new_data);
     pa_assert(u);
+
+    char* media_name = (char*) pa_proplist_gets(new_data->proplist,PA_PROP_MEDIA_NAME);
+    if(media_name!=NULL)
+    {
+        pa_log_info("hook_callback_sink_input_new %s",media_name);
+    }
+    if((media_name != NULL) && (0==strcmp(media_name,"pulsesink probe")))
+    {
+    	pa_log_info("hook_callback_sink_input_new pulsesink probe found");
+    	return PA_HOOK_OK;
+    }
```

Comment:
It would be better to describe the intetion of this code.
Because I think this is special code for pa_stream created by pulsesink pulgin for Gst.

In addition, media_name != NULL is evaluated already in previous statement.
Therefore this code can be into it.